apps/settings/js/bluetooth.js

```diff
@@ -63,7 +64,7 @@ navigator.mozL10n.once(function bluetoothSettings() {
   // device information
   var gMyDeviceInfo = (function deviceInfo() {
     var visibleItem = document.getElementById('device-visible');
-    var visibleName = document.getElementById('bluetooth-device-name');
+    var visibleName = document.querySelector('.bluetooth-device-name');
     var visibleCheckBox = document.querySelector('#device-visible input');
```

Comment:
after apply the patch I see the `visible to all` switch is easily double tapped, it might be the cause.